MvvmCross.Droid.Support.V7.AppCompat/MvxAppCompatActivity.cs

```diff
@@ -72,12 +70,6 @@ protected override void AttachBaseContext(Context @base)
         {
             base.AttachBaseContext(MvxContextWrapper.Wrap(@base, this));
         }
-
-        public override View OnCreateView(View parent, string name, Context context, IAttributeSet attrs)
-        {
-            var view = MvxAppCompatActivityHelper.OnCreateView(parent, name, context, attrs);
-            return view ?? base.OnCreateView(parent, name, context, attrs);
-        }
     }
```

Comment:
This method only installs our custom Mvx appcompat views into `MvxAppCompatActivity` views.  Without this you be installing appcompat styled views into non appcompat activities.